amqp_influxdb/__init__.py

```diff
@@ -44,6 +45,24 @@ def __init__(self,
 
         connection_parameters = connection_parameters or {}
 
+        if isinstance(connection_parameters.get('credentials'), dict):
```

Comment:
aren't the rabbit user/password mandatory?
We should probably fail if they're missing